src/pages/Signup.svelte

```diff
@@ -51,79 +54,57 @@
               </div>
               <div>
                 <label for="user-name" class="sr-only">Username</label>
-                <input
+                <Input
                   id="user-name"
                   name="name"
-                  type="name"
                   bind:value={username}
                   autocomplete="name"
                   required
-                  class="appearance-none rounded-none relative block w-full px-3 py-2 border border-gray-300 placeholder-gray-500 text-gray-900 rounded-t-md focus:outline-none focus:ring-indigo-500 focus:border-indigo-500 focus:z-10 sm:text-sm"
                   placeholder="Your name" />
               </div>
               <div>
                 <label for="password" class="sr-only">Password</label>
-                <input
+                <Input
                   id="password"
                   name="password"
                   type="password"
                   bind:value={password}
                   autocomplete="current-password"
                   required
-                  class="appearance-none rounded-none relative block w-full px-3 py-2 border border-gray-300 placeholder-gray-500 text-gray-900 rounded-b-md focus:outline-none focus:ring-indigo-500 focus:border-indigo-500 focus:z-10 sm:text-sm"
                   placeholder="Password" />
               </div>
             </div>
 
             <div class="flex justify-end">
               <div class="text-sm">
                 <a
-                  href="javascript.void()"
+                  href="/forgot-password"
+                  use:link
                   class="font-medium text-gray-500 hover:text-indigo-500">
                   Forgot your password?
                 </a>
               </div>
             </div>
 
-            <div>
-              <button
-                type="submit"
-                class="group uppercase relative w-full flex justify-center py-2 px-4 border border-transparent text-sm font-medium rounded-md text-white bg-indigo-600 hover:bg-indigo-700 focus:outline-none focus:ring-2 focus:ring-offset-2 focus:ring-indigo-500">
-                <span class="absolute left-0 inset-y-0 flex items-center pl-3">
-                  <!-- Heroicon name: lock-closed -->
-                  <svg
-                    class="h-5 w-5 text-indigo-500 group-hover:text-indigo-400"
-                    xmlns="http://www.w3.org/2000/svg"
-                    viewBox="0 0 20 20"
-                    fill="currentColor"
-                    aria-hidden="true">
-                    <path
-                      fill-rule="evenodd"
-                      d="M5 9V7a5 5 0 0110 0v2a2 2 0 012 2v5a2 2 0 01-2 2H5a2 2 0 01-2-2v-5a2 2 0 012-2zm8-2v2H7V7a3 3 0 016 0z"
-                      clip-rule="evenodd" />
-                  </svg>
-                </span>
-                create my account
-              </button>
-            </div>
+            <Button>Create my account</Button>
```

Comment:
@sumitdhungana14 Let's use this text as a prop to our button component as well.

Reply:
slot would be a better use case here IMO.